scipy/interpolate/polyint.py

```diff
@@ -439,6 +439,23 @@ def approximate_taylor_polynomial(f,x,degree,scale,order=None):
     Choosing order somewhat larger than degree may improve the higher-order
     terms.
 
+    Examples
+    --------
+    We can calculate Taylor approximation polynomials of sin function with
+    various degrees:
+
+    >>> import matplotlib.pyplot as plt
+    >>> from scipy import interpolate
+    >>> x = np.linspace(-10.0, 10.0, num=100)
+    >>> plt.plot(x, np.sin(x), label="sin curve")
+    >>> for degree in np.arange(1, 15, step=2):
+    >>>     sin_taylor = interpolate.approximate_taylor_polynomial(np.sin, 0,
```

Comment:
This should start with `...` instead of `>>>`.  Also, the line can be shortened if we use

    from scipy.interpolate import approximate_taylor_polynomial

above, and then drop `interpolate.` from the name here.

Reply:
I fixed.